crates/lib/src/command/supervisor.rs

```diff
@@ -298,6 +263,28 @@ impl Supervisor {
 		})
 	}
 
+	/// Spawns the command set, the supervision task with a random [`SupervisorId`] and returns a new control object.
+	pub fn spawn(
```

Comment:
now that we're doing breaking changes, we can remove this old one, and rename the new `new()` back to `spawn()` (just because I liked that name better here)

Reply:
bahaha, fair enough!! :D

Reply:
for the parameter to `spawn()`, would you prefer
```rust
Supervisor::spawn(args: impl Into<Args>) -> ...
```
for making potentially changing the `args` in the future easier without having to make breaking changes, or just use a concrete type that would then require either `#[non_exhaustive]` and could be constructed as a literal unless it had a `Default::default` implementation, but would be way less messy and ridiculous?

i think, but do not want to make this assumption and thought it be better to make sure, that in the past you favored the second option? this could be me misremembering what you said though!

Reply:
The second but let's not mark it as non-exhaustive so it can be constructed directly. I'll punt the question of how to handle further changes to future-me!

Reply:
okie, awesome! best of luck to future you with this worry ! :D